.github/scripts/bounties.js

```diff
@@ -0,0 +1,57 @@
+const { Octokit } = require("@octokit/core");
+const octokit = new Octokit({ auth: process.env.GITHUB_TOKEN });
+
+async function run() {
+  const githubContext = JSON.parse(process.env.GITHUB_CONTEXT);
+  const issueID = githubContext.event.issue.number;
+  const issueTitle = githubContext.event.issue.title;
+  const repoName = githubContext.event.repository.name;
+  const repoOwner = githubContext.event.repository.owner.login;
+
+  try {
+    switch (githubContext.event.action) {
+      case 'opened':
+        const body = githubContext.event.issue.body +
+          "\n\n<br /><hr>\nCare about this issue? Want to get it " +
+          "resolved sooner? If you are a " +
+          "<a href='https://www.mautic.org/become-a-member-of-mautic'>member " +
+          "of Mautic</a>, you can add some funds to the " +
```

Comment:
I suggest we make this available as a perk to Mautic members - it's an incentive to become a member, and I feel that we'd like to encourage people to support the project as a whole before throwing money at individual issues - but open for discussion (probably best to have that in the Debate on the Community Portal).